ts/packages/node/src/stores/Keyv.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable spellcheck/spell-checker */
+import { KeyValueConfigStore } from '@configu/ts';
+import type Keyv from 'keyv';
+
+export type KeyvConfigStoreConfiguration = { keyvInstance: Keyv };
+
+/**
```

Comment:
Nice work on the doc string 🥇

Reply:
Thanks! I thought it would be good to have a example popping out from the IDE while typing